<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sea Battle 2.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="player1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>422</width>
      <height>422</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>422</width>
      <height>422</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>422</width>
      <height>422</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>422</width>
      <height>422</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>420.000000000000000</width>
      <height>420.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QGraphicsView" name="player2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>422</width>
      <height>422</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>422</width>
      <height>422</height>
     </size>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>420.000000000000000</width>
      <height>420.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Placement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="actionGame_over"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="actionCreator"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCreator">
   <property name="text">
    <string>Creator</string>
   </property>
  </action>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGame_over">
   <property name="text">
    <string>Game over</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Button1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AutoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ReadyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AutoClicked()</slot>
  <slot>ReadyClicked()</slot>
 </slots>
</ui>
